envs/example/standard/group_vars/all.yml

```diff
@@ -26,3 +26,58 @@ manage_disks:
 #      filesystem_opts:
       mount_point: /mnt/test
 #      mount_opts:
+
+monitoring:
+  rabbit:
+    host: 172.16.0.103
+    user: sensu
+    pass: m2KNhjrmxgjXB2ue
+    vhost: /sensu
```

Comment:
Yep, since I removed it from here: https://github.com/blueboxgroup/ursula/pull/1959/files/cd0c1139ed1da64846251d4b56e5e9d2dab5d5f2#diff-2863e1df0e26fa52812ca98d3a2a3318L8

This is a config that works with our backend, it's not a production secret by any means. Just appears to be. Same with the cert below this.